src/django/api/templates/mail/claim_facility_revocation_body.html

```diff
@@ -21,16 +21,17 @@
                 Facility URL: {{ facility_url }}
             </li>
         </ul>
-        {% if revocation_reason|length %}
         <p>
             Here's the reason for it was revoked:
         </p>
         <p>
-            {{ revocation_reason }}
+            We have discovered your claim is actually inaccurate. If you believe your claim should
```

Comment:
This change matches what was requested in the issue, but I want to confirm that there was some discussion about not having the reason entered into the approval form appear in the email. 

The revocation form now incorrectly says that the content will be mailed to the claimant.

<img width="576" alt="Screen Shot 2019-06-28 at 10 06 59 AM" src="https://user-images.githubusercontent.com/17363/60358950-7d68c480-998c-11e9-88b7-86b740c94904.png">